integration/examples/hot-reload/node/Dockerfile

```
@@ -1,5 +1,9 @@
-FROM gcr.io/k8s-skaffold/nodemon
+FROM node:10.15.3-alpine
 
+WORKDIR /app
 EXPOSE 3000
-CMD ["nodemon","--legacy-watch", "server.js"]
-COPY src ./
+CMD ["npm", "run", "dev"]
+
+COPY package* ./
+RUN npm install
+COPY src .
```

```suggestion
COPY src src/
```
Otherwise docker flattens the directory at the destination. And in the `package.json` the main file is `src/index.js`. I think it only works by accident, because node looks for `index.js` in the root folder if the entrypoint is not found.

---

I think the idea was to flatten the directory. in the skaffold.yaml, sync is configured to strip the `src/` prefix.
I'll fix the `package.json` though.